model/delivery/factory/LtiDeliveryFactory.php

```diff
@@ -89,7 +89,7 @@ public function create(
         }
 
         $eventManager = $this->getServiceLocator()->get(EventManager::SERVICE_ID);
-        $eventManager->trigger(new DeliveryCreatedEvent($deliveryResource->getUri()));
+        $eventManager->trigger(new DeliveryCreatedEvent($deliveryResource));
```

Comment:
why are we not using the URI and sending the full resource ?

Reply:
This is consequence of [this](https://github.com/oat-sa/extension-tao-delivery-rdf/commit/593f27c24e7c11bbcef62e9adf3a5b6dea7f4c1f) change:

Reply:
Wow, so this change was done, but this very repository was not updated?

Reply:
@bartlomiejmarszal  could you bump dependency then, to not forgot do that later on integration branch merge. Should be `12.0.0` for deliveryRdf

Reply:
dependency updated